### PR TITLE
Copied files from explorer to git

### PR DESCRIPTION
**Pull request by ozzy-hga**

### CODE REVIEW

File: css/stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+  color: darkslategrey;
+}
+
+p.challenge{
+  background: palegoldenrod;
+  color: black;
+}
+
+.myDiv{
```
Comment (xdossantos) [inline css/stylesheet.css:33]: This is not specific enough - please give a more descriptive name to the class

File: css/stylesheet5.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+.ImageCircle {
+    border-style: solid;
+    border-color: pink;
+    border-width: 3.5px;
+    border-radius: 200px 200px 200px 200px;
+}
+
+.PadIt {
```
Comment (xdossantos) [inline css/stylesheet5.css:45]: Do not start class names with an upper case

File: css/stylesheet6.0.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+aside{
+    background-color: #E9D985;
+    position: relative;
+    padding-left: 20px;
+    margin-top: 0px;
+    flex: 1;
+    margin-bottom: 0px;
+}
+
+.sideMenu{
```
Comment (xdossantos) [inline css/stylesheet6.0.css:42]: Great job on using camel case

File: css/stylesheet.css (hunk 1 above)
Comment (ozzy-hga) [reply css/stylesheet.css:33]: Noted
Will make changes

**Pull request merged**